torch/onnx/symbolic_opset16.py

```diff
@@ -0,0 +1,46 @@
+# EDITING THIS FILE? READ THIS FIRST!
+# see Note [Edit Symbolic Files] in symbolic_helper.py
+
+# This file exports ONNX ops for opset 16
+
+# Note [ONNX Operators that are added/updated in opset 16]
+#
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# https://github.com/onnx/onnx/blob/main/docs/Changelog.md#version-16-of-the-default-onnx-operator-set
+# New operators:
+#   GridSample https://github.com/onnx/onnx/pull/3557
+#
+# Updated operators:
+#    Identity
+#    If
+#    LeakyRelu
+#    Loop
+#    PRelu
+#    RoiAlign
+#    Scan
+#    ScatterElemenets
+#    ScatterND
+#    Where
+#    GreaterOrEqual
+#    LessOrEqual
+#    SequenceMap
+
+from torch.onnx.symbolic_helper import parse_args
+
+from torch.nn.functional import GRID_SAMPLE_INTERPOLATION_MODES, GRID_SAMPLE_PADDING_MODES
+
+
+# note (mkozuki): Why `grid_sampler` instead of `grid_sample`?
```

Comment:
nit:

```suggestion
# NOTE(crcrpar): Why `grid_sampler` instead of `grid_sample`?
```

Reply:
I would add this to the docstring instead.

Reply:
can't that be a follow-up thing? I didn't see any docstring in symbolic_opset files.

Reply:
It can be! Totally non blocking